packages/components/src/tooltip/index.js

```diff
@@ -214,10 +240,12 @@ function Tooltip( props ) {
 		: getRegularElement;
 
 	const popoverData = {
+		anchorRef: childRef,
 		isOver,
+		offset: 4,
```

Comment:
Why we need this offset to be `4`?

Reply:
I was a bit unsure of this one, too. Prior to this PR the offset is set to `12`, I think probably to account for the positioning not being correct. With this PR, now that the Tooltip is anchored to the element, the Tooltip lines up right next to the element, so I wasn't sure if it should have some breathing room. Perhaps it's better for us to remove the `offset` in this PR, and then in a potential follow up we could expose an `offset` option for Tooltips that want to add more room?

Reply:
Here's the difference between offset `4` and leaving it `undefined`:

| Offset set to 4 | No offset |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/14988353/178632325-71815e50-3766-433d-a9dd-e4f843733ab9.png) | ![image](https://user-images.githubusercontent.com/14988353/178632361-62a62d2e-56ed-4869-be0b-5c37ab6b9426.png) |

After testing removing the offset, I kinda prefer the extra breathing room, but happy to get rid of it if it feels too opinionated!

Reply:
I think we can keep this for now too.